scala/src/main/scala/smile/clustering/distance/Binary/ShapeDifference.scala

```diff
@@ -0,0 +1,14 @@
+package smile.math.distance
+
+import scala.math.pow
+/**
+ * @author Beck Gaël
+ *
+ **/
+class ShapeDifference extends BinaryDistance {
```

Comment:
Please add comment what this is, how it is computed, and what use cases it is good for.

Reply:
I don't know good use case, i just implemented it in order to test them. Did i need to remove them ?

Reply:
Do you just create ShapeDifference and PatternDIfference by yourself? If not, any reference?

Reply:
[A survey of binary measurement.pdf](https://github.com/haifengl/smile/files/1777259/A.survey.of.binary.measurement.pdf)
I found these measures ih that survey.